sktm/patchwork.py

```diff
@@ -644,13 +644,16 @@ def _set_patch_check(self, patch, payload):
             patch:      JSON representation of a patch to add the check for.
             payload:    The "check" payload dictionary to be converted to JSON.
         """
-        r = requests.post(patch.get("checks"),
-                          headers={"Authorization": "Token %s" % self.apikey,
-                                   "Content-Type": "application/json"},
-                          data=json.dumps(payload))
+        response = requests.post(
+            patch.get("checks"),
+            headers={"Authorization": "Token %s" % self.apikey,
+                     "Content-Type": "application/json"},
+            data=json.dumps(payload)
+        )
 
-        if r.status_code not in [200, 201]:
-            logging.warning("Failed to post patch check: %d" % r.status_code)
+        if response.status_code not in [200, 201]:
+            logging.warning("Failed to post patch check: %d",
+                            response.status_code)
```

Comment:
It's great that you mention the extra things tucked into the commit, and this is a small one, but I think it would still be better to keep them separate :) Not required for my ACK on this, though.